container/lxd/network.go

```diff
@@ -138,9 +151,9 @@ func (s *Server) ensureDefaultBridge(profile *api.Profile, eTag string) error {
 	return errors.Trace(s.UpdateProfile(profile.Name, profile.Writable(), eTag))
 }
 
-// ensureNoIPv6 checks that the input network has no IPv6 configuration.
+// verifyNoIPv6 checks that the input network has no IPv6 configuration.
 // An error is returned when it does.
-func ensureNoIPv6(net *api.Network) error {
+func verifyNoIPv6(net *api.Network) error {
```

Comment:
I'd love it if we could just support IPv6 rather than forcing it to not exist.
I would guess the hard problem is 'dual stack' moreso than v6 or v4